aspnetcore/signalr/hub-filters.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Use hub filters in SignalR for ASP.NET Core
+author: brecon
+description: Learn how to use hub filters in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-5.0'
+ms.author: brecon
+ms.custom: mvc
+ms.date: 05/22/2020
+no-loc: [Blazor, "Identity", "Let's Encrypt", Razor, SignalR]
+uid: signalr/hub-filters
+---
+
+# Use hub filters in SignalR for ASP.NET Core
```

Comment:
Should hub be capitalized? cc @scottaddie

Reply:
I don't think we use title case doc section headers. If we capitalize "hub" here, we'd need to update all our other SignalR docs that default to lower case "hub".

Reply:
In this context, we're fine with lower case.